.gitignore

```diff
@@ -18,3 +22,4 @@ target
 nb-configuration.xml
 nbactions.xml
 
+"" 
```

Comment:
Why this "" ?

Reply:
Fixed in 4d08affe85545b26afed5461585bc8bd710d5a85